tests/main/nfs-support/task.yaml

```diff
@@ -0,0 +1,96 @@
+summary: Test that snaps still work when /home is a NFS mount
```

Comment:
:heart:

Reply:
As things go, this test obviously caught a bug in the initial code that worked fine for /home sub-directories but not the full thing :)